client/chat/action-creators.ts

```diff
@@ -55,7 +55,9 @@ export function joinChannelWithErrorHandling(
         let message = `An error occurred while joining ${channelName}`
 
         if (isFetchError(err) && err.code) {
-          if (err.code === ChatServiceErrorCode.UserBanned) {
+          if (err.code === ChatServiceErrorCode.MaximumJoinedChannels) {
+            message = "You can't join anymore channels."
```

Comment:
"any more" is what you'd want here (English sucks I know :) )

I think it might be nice to explain this a bit further though, not totally clear to me *why* I can't join more channels with this. Perhaps:

"You have reached the limit of joined channels. You must leave one before you can join another."